src/screens/SendAddressInput.js

```diff
@@ -5,73 +5,74 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React from 'react';
+import React, { useState } from 'react';
 import { KeyboardAvoidingView, View } from 'react-native';
 import { t } from 'ttag';
+import { Network } from '@hathor/wallet-lib';
+import { useSelector } from 'react-redux';
+import { useNavigation, useRoute } from '@react-navigation/native';
 
 import NewHathorButton from '../components/NewHathorButton';
 import SimpleInput from '../components/SimpleInput';
 import HathorHeader from '../components/HathorHeader';
 import { getKeyboardAvoidingViewTopDistance, validateAddress } from '../utils';
 import OfflineBar from '../components/OfflineBar';
 
-class SendAddressInput extends React.Component {
+export const SendAddressInput = () => {
+  const route = useRoute();
+  const navigation = useNavigation();
   /**
    * address {string} send tokens to this address
    * error {string} address validation error
    */
-  constructor(props) {
-    super(props);
-    this.state = {
-      // we can optionally receive a string to fill out the address
-      // input (for eg, user scanned QR code)
-      address: this.props.route.params?.address ?? null,
-      // TODO this is probably temporary. We don't have the UI for error message yet.
-      error: null,
-    };
-  }
+  const [formModel, setFormModel] = useState({
+    // we can optionally receive a string to fill out the address
+    // input (for eg, user scanned QR code)
+    address: route.params?.address ?? null,
+    // TODO this is probably temporary. We don't have the UI for error message yet.
```

Comment:
Should we remove this? It seems that we are using this information and showing error messages.

Reply:
Removed: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/376/commits/e3204a86ad155033a0002f20c8393ec41357d407